apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var ConversationAccessor = require('./accessors');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var requireFromApp = require(appRoot + '/shared/test/integration/require_from_app');
+
+function Conversation(client) {
+  this.client = client;
+  this.accessors = ConversationAccessor;
+}
+module.exports = Conversation;
+
+Conversation.prototype.callContact = function() {
+  this.accessors.callButton.tap();
+
+  var Dialer = requireFromApp('dialer', 'lib/dialer');
```

Comment:
I still don't like that we use ```require``` here, I'll try to think more to see if I can come up with better solution :)